src/services/directory.ts

```diff
@@ -1,8 +1,9 @@
-import { Member } from '../types';
+import { Member, RawMemberData } from '../types';
 import api from './api';
 import { devPrint } from '../components/utils/RandomUtils';
+import { AxiosResponse } from 'axios';
 
-function deserializeMember(data: any): Member {
+function deserializeMember(data: RawMemberData): Member {
```

Comment:
You can also simplify this as:
```
function deserializeMember({
  user: id,
  first_name: firstName,
  last_name: lastName,
  grad_date: gradDate,
  discord_username: discordUsername,
  resume_url: resumeUrl,
  discord_id: discordId,
  ...rest
}: RawMemberData): Member {
  return {
    id,
    firstName,
    lastName,
    gradDate,
    discordUsername,
    resumeUrl,
    discordId,
    ...rest,
  };
}
```

I really like this pattern personally.

Reply:
oh yeah that looks nice. Also I'm planning on moving deserializers outside of the services module in an subsequent PR so they can be shared FWIW